Makefile

```diff
@@ -1,32 +1,43 @@
-default: ## Run app
-	@. ./venv/bin/activate && FLASK_APP=src/app FLASK_ENV=development flask run
+defualt: ## Run app
```

Comment:
Looks like you've utilized a previous version for the make operations here.
Change `defualt` to `default`.
Check commit `ad80f4759eeddc9311faeee953acfad528471ca7`.